client/src/shared/services/ajaxFactory.js

```diff
@@ -23,6 +23,15 @@
       });
     };
 
+
+    ajaxObj.isOrderComplete = function(transactionId){
+      return $http({
+        method: 'GET',
+        url: serverUrl + '/order/details',
+        params: {id: transactionId}

```

Comment:
Space after and before curly braces if they are on the same line.

`{ id: transactionId }`